### PR TITLE
Add Suave Toliman testnet

### PR DESCRIPTION
**Pull request by pistomat**
Add the network specs for Toliman testnet of the Suave network.

CC: @dmarzzz @ferranbt

### CODE REVIEW

**Comment by dmarzzz** [comment]
@0x416e746f6e does this match our config? I know we need to update currency to TEEth but wondering if you could double check the rest or @sukoneck

File: _data/chains/eip155-33626250.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Toliman Suave Testnet",
+  "chain": "ETH",
+  "rpc": ["https://rpc.toliman.suave.flashbots.net"],
+  "faucets": ["https://faucet.toliman.suave.flashbots.net"],
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
```
Comment (0x416e746f6e) [inline _data/chains/eip155-33626250.json:8]:
```suggestion
    "name": "SUAVE Toliman Eth",
    "symbol": "TEEth",
```

**Comment by 0x416e746f6e** [comment]
> @0x416e746f6e does this match our config? I know we need to update currency to TEEth but wondering if you could double check the rest or @sukoneck

@dmarzzz I left a comment w.r.t. the currency symbol indeed.

chain/network IDs and URLs are correct.

**Comment by dmarzzz** [comment]
ty @0x416e746f6e ! @pistomat and @ligi once we merge recommended change above this is good to merge, we also officially launched the network last week https://collective.flashbots.net/t/announcing-our-new-suave-testnet-toliman/3770

**Comment by ligi** [comment]
no merge if CI is failing

**Comment by pistomat** [comment]
Thanks guys for the review.

I pushed the suggested update, now the prettier part of the CI is ✅ @ligi, locally both `npx prettier` and `gradlew run` finish successfully.

File: _data/chains/eip155-33626250.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "rpc": ["https://rpc.toliman.suave.flashbots.net"],
+  "faucets": ["https://faucet.toliman.suave.flashbots.net"],
+  "nativeCurrency": {
+    "name": "SUAVE Toliman Eth",
+    "symbol": "TEEth",
+    "decimals": 18
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://suave-alpha.flashbots.net/toliman",
+  "shortName": "",
```
Comment (0x416e746f6e) [inline _data/chains/eip155-33626250.json:13]: I guess this should make linter happy => CI pass:

```suggestion
  "shortName": "suave-toliman",
```

**Review by ligi — APPROVED** [review]

**Pull request merged**